src/app/zap-templates/templates/app/helper.js

```diff
@@ -25,6 +25,7 @@ const queryEvents  = require(zapPath + 'db/query-event.js')
 const cHelper      = require(zapPath + 'generator/helper-c.js')
 const string       = require(zapPath + 'util/string.js')
 const dbEnum       = require(zapPath + '../src-shared/db-enum.js')
+const accessHelper = require(zapPath + 'generator/helper-access.js')
```

Comment:
Where is accessHelper used?

Reply:
I don't use it, but my understanding is that code needs to be loaded.

Reply:
Why?  If it's not used, why are we loading it?

Reply:
The variable isn't used, but I thought loading the JS file was necessary for the access elements in the zap templates to work properly. However, I tried it again without it, and it seems to work, so removing.